weejson/jackson/src/com/rallyhealth/weejson/v1/jackson/VisitorJsonGenerator.scala

```diff
@@ -203,7 +203,7 @@ class VisitorJsonGenerator[J](
 
   override def writeTree(rootNode: TreeNode): Unit = objectCodec.writeValue(this, rootNode)
 
-  override def getOutputContext: JsonStreamContext = throw notSupported
+  override def getOutputContext: JsonStreamContext = null
```

Comment:
throwing is bad, yes. but `null` is as bad. Would `JsonStreamContext.empty` (so you get back the expected type but it doesn't really have a context data) be creatable?

Reply:
I'd take a PR. `inRoot()`, `inObject()`, `inArray()` would all be possible to implement over `stack: mutable.ArrayStack[ObjArrVisitor[Any, _]]` as well.